dependencies.gradle

```diff
@@ -1,6 +1,6 @@
 project.ext.versions = [
     checkstyle: '7.6.1',
-    atlas: '5.1.8',
+    atlas: '5.2.3',
```

Comment:
I think you might be looking for `atlas 5.2.2`, no?

Reply:
Nope, waiting for 5.2.3 :)

Reply:
Oh, nevermind. I understand what you are doing now :)